public/editor/scripts/editor/js/fc/bramble-popupmenu.js

```diff
@@ -22,6 +22,8 @@ define(function(require) {
     self.close = function(e) {
       if(e) {
         e.stopPropagation();
+        self._button$.removeClass('active');
+        $("#editor-pane-nav-options").css("opacity", "0.5");
```

Comment:
Now that we are adding and removing the ``active`` class to the element, we can get rid of the code that changes the CSS and instead add some styles to the ``editor.less`` file [here](https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/stylesheets/editor.less#L351)

Like this...
```
#editor-pane-nav-options.active,
#editor-pane-nav-options:hover {
    opacity: 0.7;
    filter: alpha(opacity=70);
}
```